defs/bitcoin/primecoin.json

```diff
@@ -0,0 +1,38 @@
+{
+  "coin_name": "Primecoin",
+  "coin_shortcut": "XPM",
+  "coin_label": "Primecoin",
+  "website": "https://primecoin.io",
+  "github": "https://github.com/primecoin",
```

Comment:
change this to https://github.com/primecoin/primecoin

Reply:
Ok this is done